indico/modules/events/editing/models/tags.py

```diff
@@ -31,10 +31,15 @@ def __table_args__(cls):
         index=True,
         nullable=False
     )
-    name = db.Column(
+    title = db.Column(
         db.String,
         nullable=False
     )
+    code = db.Column(
+        db.String,
+        index=True,
```

Comment:
is that index really useful?

Reply:
IIRC I made the diff before I added that (?)

Reply:
My thought was that sorting by code might come in helpful in the future, when we implement the µS.

Reply:
I guess those queries would be event-scoped so even if we queried a single tag (instead of just fetching them all which at least to me sounds more likely since a "get a single tag" api sounds like overkill considering that tags are small and there won't be hundreds in an event) there would be little benefit in having this extra index...

Reply:
OK, makes sense. I'll remove that.